charts/cu-cp/templates/deployment-cu-cp.yaml

```diff
@@ -13,6 +13,12 @@ top:
 values:
   {{ $values | toYaml | nindent 2 }}
 
+selectorLabels:
+  {{ include "accelleran.common.selectorLabels" (dict "top" $ "values" $values) | nindent 2 }}
+  app.kubernetes.io/component: {{ $values.component }}
```

Comment:
Do we need a check here whether values.component exists? Or maybe we could fall back to a default? Will helm throw an error if this is not defined?

Reply:
In this case we have the default specified within the values file as the values are always merged anyway with something a user provides.